_includes/zoom-button.html

```diff
@@ -1,5 +1,7 @@
-{% if page.meeting_url %}
-<a class="btn btn--success btn--zoom" href="{{ page.meeting_url }}">
+{% capture now %}{{'now' | date: '%s' %}}{% endcapture %}
+{% capture event_end %}{{ time.end | date: '%s' }}{% endcapture %}
```

Comment:
Yes, this is why this is still a draft :)
I wanted to know if we want to do an `assign time` or if we better want to access `page.time`. As you did the assignment for the calendar I did not want to make a decision yet.

Reply:
Yep, I would use `page.time` (or rather `{% assign time = page.time | last %}`). I only assigned `time` on the kickoff page due to how the add-to-calendar-button works.

Reply:
> do we still aim at repeating events as mentioned in #275? Because then we would get another level.

Yep, but this is not yet implemented. On my to-do list for next week

Reply:
I'll update the assignment when I implement it